src/utils.js

```diff
@@ -286,6 +309,14 @@ export function changeDateFormat(timestamp) {
   return `${month} ${day}, ${year} @ ${time}`
 }
 
+//convert to snake case
+export function convertToSnakeCase(value) {
```

Comment:
I'd suggest to use an external library to do this: https://www.npmjs.com/package/snake-case. Is there is a reason to not use it...
 
which is the expected format of `value`? Can contain spaces,dashes or any other separator? Can we add a condition to check if the format is valid?

Reply:
I believe this function is dead code, I don't see it being used anywhere, so let's remove it?